### ISSUE TITLE
Question: How to increase the test's timeout?

### ISSUE THREAD

**Issue by nikodamnCH** [vitalets/playwright-bdd#89]
Hey! Firstly, thank you for creating and maintaining this repository. It's really super useful.

I have a question regarding increasing the timeout of just some tests. Of course there's a way to set timeout in the Playwright's global configuration, but I'm looking for a way to increase it only for some specific tests. Playwright doc mentions the `test.setTimeout()`, but it's not available in the BDD Playwright obviously. Are you aware of any solution to increase the timeout for specific tests or in a particular BDD step?

**Comment by vitalets** [comment]
Hi @nikodamnCH !

The easiest way to increase timeout for some tests is to mark them with some tag (e.g. `@slow`) and then set timeout in a fixture. For example:
```gherkin
Feature: Todo Page
    
    @slow
    Scenario: Empty list
      Given I am on todo page
      Then visible todos count is 0
```
Fixtures:
```ts
export const test = base.extend<{ slowTests: void }>({
  slowTests: [async ({ $tags }, use, testInfo) => {
    if ($tags.includes('@slow')) testInfo.setTimeout(5000);
    await use();
  }, { auto: true }],
});
```
-- 
If you want to increase test timeout in the particular step (when you know that this step takes a long time), you can do it right in step definition using `$test` fixture that represents Playwright's `test` object:
```ts
Given('I am on Playwright home page', async ({ $test }) => {
  $test.setTimeout(5000);
  // ...
});
```

**Comment by nikodamnCH** [comment]
Great stuff, thanks a lot!

**Comment by vitalets** [comment]
You are welcome :)

**Comment by NikkTod** [comment]
Hi @vitalets,  I am having the same issue with the timeout. 
Using Decoractors syntax can you please share an example how can we use the .setTimeout() in a particular step ?

The below does not work for me:

```
import {page, expect} from 'playwright/test'
import {Fixture, Given, When, Then} from 'playwright-bdd/decorators'
import {test} from 'base/fixtures'

Given('A step that needs a bit more timeout')
async someStep() {
  test.setTimeout(50000);
  //...
}
```

**Comment by vitalets** [comment]
@NikkTod 
Your code sample works for me. I've created a repro [here](https://github.com/vitalets/playwright-bdd-example/tree/decorators-test-timeout). Let me know if it does not work.

Notice that we set timeout of the whole test, not a particular step. For now Playwright [does not have](https://playwright.dev/docs/api/class-test#test-step) option to set timeout of the step. This could be a feature request for Playwright team!
